build/buildrelease.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$1" = "" ]]
+then
+  echo "Usage: buildrelease.sh version"
+  echo "e.g. buildrelease.sh 2.0.0"
+  echo "or buildrelease.sh 2.0.0-rc1"
+  echo "It is expected that a git tag will already exist"
+  exit 1
+fi
+
+declare -r VERSION=$1
+declare -r SUFFIX=$(echo $VERSION | cut -s -d- -f2)
+declare -r BUILD_FLAG=$([ "$SUFFIX" != "" ] && echo "--version-suffix $SUFFIX")
+declare -r OUTPUT=artifacts
+
+rm -rf releasebuild
```

Comment:
Again, if you're going to assume a particular starting directory, please put something like

```sh
cd $(dirname $0)/..
```

near the top of the script.